### DISCUSSION TITLE
joinFieldPath method

### DISCUSSION THREAD

**Discussion by nguyenvantruong17** [gialynguyen/vcc-schema#5] [Q&A] [answered]
```tsx
export const joinFieldPath = (paths: string[]) =>
  paths.join(".").replace(/\.\[/g, "[");
```
Hàm này dùng để làm gì nhỉ?
Mình có nên mô tả input output ở discussion này để sau này update nó thành 1 issue không?

**Comment by gialynguyen** [comment, marked as answer]
khi duyệt vào bên trong object thì paths là 1 biến để tracking lại "đường đã đi qua" khi duyệt. 
VD:
```
{
     address: {
         age: 12
     }
}
```
Thì khi ta duyệt đến `address.age` thì paths lúc này sẽ là ['address', 'age'], thì hàm `joinFieldPath` sẽ transfer dạng mảng thành dạng string như ban đầu `address.age`.

```
Input: ['address', 0, 'name']
Output: address[0].name
```

**Reply by gialynguyen** [reply]
@nguyenvantruong17 This has changed. Now, It like this:
![image](https://user-images.githubusercontent.com/29276005/120336056-31fb8580-c31c-11eb-9153-e2edc98845ab.png)
